### DISCUSSION TITLE
Failed to Execute Inline Script: one or more of the requirements were not found.

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#911] [Q&A] [answered]
I notice in the meta.targets, all the examples represent an element ID. 

As you will see here, I am trying to reference a wave component by class. My script is failing and I am getting a console log error in the browser saying "Failed to execute inline script: one or more of the requirements were not found." I know last time this was because I was referencing a class instead of ID, which I still am, but thought this was supported with 0.17! This is how I am referencing the elements' class in meta.
```   
q.page['meta'] = ui.meta_card(box='', script=ui.inline_script(
        content = col_script,
        targets = ['ms-DetailsHeader-cellName cellName-173'],

```
Any thoughts would be helpful!

![image](https://user-images.githubusercontent.com/78372056/125140840-0b79f880-e0d9-11eb-979f-54105e53b3a2.png)

**Comment by lo5** [comment, marked as answer]
You probably want `'.ms-DetailsHeader-cellName .cellName-173'`, which is interpreted as "look for an element with class name `cellName-173` inside an element with class name `ms-DetailsHeader-cellName`".

On the other hand, if `cellName-173` is an ID and not a class name, use `'.ms-DetailsHeader-cellName #cellName-173'`.

In general, prefix IDs with `#` and class names with `.`. See [selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors#basic_selectors).

**Reply by Alex-Wenner-FHR** [reply]
Thank you!
